Makefile

```diff
@@ -0,0 +1,53 @@
+DIRS     ?= $(shell go list ./...)
+PKG_NAME ?= goyagi
+
+COVERAGE_PROFILE ?= coverage.out
+
+GOTOOLS := \
+	github.com/alecthomas/gometalinter \
```

Comment:
Something just to call out in terms of keeping Goyagi up to date -- we'll need to look into replacing gometalinter with golangci-lint across our Go repos / in Goyagi since it's deprecated 😬

Reply:
Yeah.. I noticed that. I added it to my list of things to bring up :D 
I personally use go vet.